packages/semaphore/src/main.nr

```diff
@@ -4,8 +4,7 @@ use edwards::ScalarField;
 use std::{hash::poseidon};
 use lean_imt::lean_imt::{LeanMerkleTree, Calculator};
 
-// TODO: change this to the actual max depth
-global MAX_DEPTH: u32 = 16;
+global MAX_PATH_LENGTH: u32 = 256;
```

Comment:
This causes an error because the bBN256 base field is F_p where p = 2^254 + 21559.
```
[semaphore.nr] Testing test_main ... FAIL
Failed to solve program: 'Cannot satisfy constraint'

error: Failed constraint
   ┌─ std/field/mod.nr:68:20
   │
68 │             assert(bits.len() <= p.len());
   │                    ---------------------
   │
   = Call stack:
     1. src/main.nr:84:29
     2. src/main.nr:37:27
     3. /Users/hyunminlee/Development/modori/semaphore.nr/packages/lean_imt/src/lean_imt.nr:44:35
     4. std/field/mod.nr:68:20
```
https://github.com/noir-lang/noir/blob/master/noir_stdlib/src/field/mod.nr#L68